### PR TITLE
[MMSYS] Prevent the same sound from being added multiple times

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Proposed changes
- Check if the list already contains the sound

Before:
ros_mmsys_before

After:
ros_mmsys_after

### CODE REVIEW

File: dll/cpl/mmsys/sounds.c [hunk 1]
```diff
@@ -1182,7 +1209,13 @@ SoundsDlgProc(HWND hwndDlg,
 
                     if (GetOpenFileNameW(&ofn))
                     {
-                        // FIXME search if list already contains that sound
+                        // search if list already contains that sound
+                        lResult = FindSoundFileInList(hwndDlg, filename);
+                        if (lResult != CB_ERR)
```
Comment (learn-more) [inline dll/cpl/mmsys/sounds.c:1214]:
```suggestion
                        if (lResult >= 0)
```
Seems a bit more intuitive, but fine either way.

**Review by learn-more — APPROVED** [review]

File: dll/cpl/mmsys/sounds.c (hunk 1 above)
Comment (TAN-Gaming) [reply dll/cpl/mmsys/sounds.c:1214]: I think `CB_ERR` is more readable.

**Comment by JoachimHenze** [comment]
Please perform the following test before committing:
Make sure that the combobox is completely empty. Then try to add a first entry via the browse button.
Does that still work after this patch?

In case it wouldn't work yet, then please fix it before merging!

**Comment by TAN-Gaming** [comment]
It still works after this PR applied.

[ReactOS-screen0.webm](https://user-images.githubusercontent.com/86486978/203904894-72d89326-22ab-47ae-bd2e-3ac45a3e8a69.webm)

**Review by JoachimHenze — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]

**Pull request merged**